sdk/core/CMakeLists.txt

```
@@ -13,6 +13,11 @@ if (NOT DISABLE_AZURE_CORE_OPENTELEMETRY)
   add_subdirectory(azure-core-tracing-opentelemetry)
 endif()
 
+# AMQP is an optional feature, don't include it unless explicitly enabled.
+if (ENABLE_AZURE_CORE_AMQP)
+add_subdirectory(azure-core-amqp)
```

nit: add indentation
```suggestion
  add_subdirectory(azure-core-amqp)
```